portfolio/src/main/java/com/google/sps/servlets/CheckStatus.java

```diff
@@ -1,13 +1,28 @@
-WebServlet("/status")
+package com.google.sps.servlets;
+
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/status")
 public class CheckStatus extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
+    response.setContentType("application/json;");
     PrintWriter out = response.getWriter();
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
-        out.println("<p>Logged in as " + userService.getCurrentUser().getEmail() + "!</p>");
-    else{
-        out.println("<p>Currently logged in as Guest.</p>");
-    }
\ No newline at end of file
+        out.println(new Gson().toJson(1));
```

Comment:
If you're using 1 or 0 to mark a user as logged in, wouldn't it be more intuitive to use a boolean instead?

`out.println(new Gson().toJson(true));`

Reply:
+1 if this is possible